orchestrator/workflows/steps.py

```diff
@@ -206,10 +207,12 @@ def _cache_other_subscriptions(product_block: ProductBlockModel) -> None:
     for subscription_id in cached_subscription_ids:
         subscription_model = SubscriptionModel.from_subscription(subscription_id)
         to_redis(build_extended_domain_model(subscription_model))
+        sync_broadcast_invalidate_cache(str(subscription.subscription_id))
```

Comment:
So, are we only sending a UUID to the frontend? How does the frontend know it's about a subscription?

Reply:
yea, good spot...

changed to `subscription:uuid` and also changed process to `process:uuid`